doc/source/security/encrypt-traffic.rst

```diff
@@ -134,12 +180,15 @@ The following files are required:
   These files are required only if the node should act as a MySQL client.
   For example, if you are planning to perform SST using ``mysqldump``.
 
+.. note:: :ref:`upgrade-certs` subsection covers the details on upgrading
+   certificates, if necessary.
+
 .. _enable-encryption:
 
-Enabling Encryption
-===================
+Enabling Encryption Manually
+============================
 
-To enable encryption, you need to specify the location
+To enable encryption manually, you need to specify the location
```

Comment:
In this section we are introducing a new term name ":ref:`encrypt-client-server`" ... but then following text re-name it as "Encrypting Database Traffic"...

Also, encrypting database traffic is more generic ... the said setting is only encrypting client-server communication.

Reply:
Move the SST section below IST section.

Reply:
So in this case it's better to make such a note, and remove the "manually" word from the "Generating Keys and Certificates Manually" title (user may need information from this section to populate auto-generated certs among all nodes).

Reply:
I've looked through this section once again and this comment looks unclear: the section contains no mentions of database traffic encryption anymore, and SST subsection is placed after encrypting replication traffic / IST, so I don't see what else can be fixed here.